### ISSUE TITLE
Improve "chat experience" for new users

### ISSUE THREAD

**Issue by stpaultim** [backdrop-ops/backdropcms.org#660]
Recently the PMC voted to change the official chat channel from Gitter to Zulip. However, in our initial weeks of activity in Zulip, at last two active members of our community have expressed frustration at the decision and at their experience in Zulip. 

My intention is to open this issue to discuss specific problems with our "chat" experience and look for solutions/improvements. If there are frustrations with the decision, I would prefer that we address them (if possible). 

The need for this discussion might go away or get larger, as we spend more time using Zulip. 

There is little debate that the user interface for Zulip is at minimum a bit confusing to newcomers. 

However, my understanding is that some users consider the UI more than a minimal inconvenience and potentially a real problem for onboarding and welcoming new members into our community. 

I suggest that we do the following:

1) We keep an open discussion going about the weaknesses of Zulip and do what we can to address them through on-boarding, configuration, and modeling good use of the tool. Also - posting issues/ideas to the Zulip project issue queue.

2) I don't want to change platforms again anytime soon, but I do think we should keep an open mind and revisit this decision in 6-9 months after we have properly tested Zulip as a community and fully exposed it's weaknesses AND taken advantage of it's strengths. While making a change was a big deal, I would be willing to revisit the decision eventually (but, I think now is too soon). 

3) Keep an open mind about using more than one tool. For the last two weeks, I've both kept an eye on Zulip and Gitter. I would not be 100% against keeping Gitter open as a place for complete newcomers to drop in and ask questions - while maintaining Zulip as the place were the community gets work done. I know that is not ideal, but I think it's worth considering. 

4) Listen to users and continue to gather feedback on the Zulip experience. 

In summary, I'd like for us to acknowledge the concerns that have been raised and try to make sure that we address them in whatever way makes the most sense, while respecting the decision that we made just 2 weeks ago.

**Comment by klonos** [comment]
I have already started posting some feedback on issues in the Zulip issue queue on GitHub - since this is an open-source project, I expect it to improve over time.

Re point 3. I would like to eventually have less places to monitor activity. I have not seen many comments in Gitter lately anyway, so I think that the community has shifted focus to Zulip already. Perhaps we can make it so that if anyone posts something on our Slack or Gitter, there is a topic in Zulip that notifies us about these events ...that way we can still hop over and help people, w/o having to actively keep an eye on these 2 backup chat platforms.

**Comment by ghost** [comment]
I haven't heard any negative feedback about Zulip recently, and we've had a few newcomers in that time too. I recommend closing this issue now.
